index.html

```diff
@@ -763,6 +769,79 @@ <h2>Creating a PR</h2>
+          <li>`names` - This property is used to set multiple names to the command. If you are using `name` don't use this.</li>
+          <li>
+            `description` - This property is used to set the description of the command; in other words,
+            to describe what the command will do when its called. This will show into the `help` command.
+          </li>
+          <li>
+            `parameters` - (Optional) - This property is used to define the available parameters of the command,
+            this will show into the `help` command. To define an optional parameter use: `&lt;?name-of-parameter&gt;`,
+            or use `&lt;name-of-parameter&gt;` if the parameter is required; when a parameter only has some specific
+            values use `&lt;action (action-1/action-2)&gt;`.
+          </li>
+          <li>
+            `run` - This property is the actioner of the command, in this you set what the command will do when its called.
+          </li>
+        </ul>
+        <h2>Creating a message repeater command</h2>
+        <ul>
+          <li>
+            First, go to `./src/commands/general/` and create a new file called `messageRepeater.ts` or as you wish.
+          </li>
+          <li>
+            Then, add the basic structure of our command:<br>
+            <pre><code style="white-space: normal;">
+              import CommandInt from "@Interfaces/CommandInt";<br><br>
+              const messageRepeater: CommandInt = {<br>
+                name: "messageRepeater",<br>
+                description: "Send a message to the current channel using the bot.",<br>
+                parameters: ["&lt;message&gt;: Message to send."],<br>
+                run: async (message) => {<br>
+                  console.log("Message repeater command called!");<br>
+                },<br>
+              };<br><br>
+              export default messageRepeater;
+            </code></pre>
+            Save the file, start the bot (You can use `npm run dev` instead of `npm run build && npm start`) and then
+            in the chat write `{prefix}messageRepeater` and in the console you will see `Message repeater command called!`.            
+          </li>
+          <li>
+            So, to this moment you have created a simple command that shows you a message in the console, but it does not
+            do what we want.<br><br>
+            First, add `const { channel, commandArguments } = message;` instead of the `console.log`, with this new line you will get 
+            the arguments of the command; in other words, the content of the message split by whitespaces and ignoring the
+            first element (The first element is the command name); and the channel of the message.<br><br>
+            Now, add `const msg = commandArguments.join(" ");` to join the arguments with whitespaces to get the
+            parameter that we require (Message).
```

Comment:
```suggestion
            First, replace the `console.log` line with `const { channel, commandArguments } = message;`. With this you will get 
            the arguments of the command, or the content of the message split by whitespaces and ignoring the
            first element (The first element is the command name), and the channel of the message.
            Now, add `const msg = commandArguments.join(" ");` to join the arguments with whitespaces to get the
            parameter that we require (Message).
```
Verbiage. Also, `<br>` tags are inaccessible and do not work well with screen readers, so should be avoided.